File: seed/csharp-sdk/exhaustive/no-root-namespace-for-core-classes/src/SeedApi.DynamicSnippets/Example115.cs
using SeedApi;
using SeedApi.Core;

namespace Usage;

public class Example115
{
    public async Task Do() {
        var client = new SeedApiClient(
            token: "<token>",
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.Reqwithheaders.GetwithcustomheaderAsync(
            new ReqWithHeadersGetWithCustomHeaderRequest {
                TestEndpointHeader = "X-TEST-ENDPOINT-HEADER",
                Body = "string"
            }
        );
    }

}

File: seed/java-sdk/exhaustive/local-files/src/main/java/com/snippets/Example115.java
package com.snippets;

import com.fern.sdk.SeedApiClient;
import com.fern.sdk.resources.reqwithheaders.requests.ReqWithHeadersGetWithCustomHeaderRequest;

public class Example115 {
    public static void main(String[] args) {
        SeedApiClient client = SeedApiClient
            .builder()
            .token("<token>")
            .url("https://api.fern.com")
            .build();

        client.reqwithheaders().getwithcustomheader(
            ReqWithHeadersGetWithCustomHeaderRequest
                .builder()
                .testEndpointHeader("X-TEST-ENDPOINT-HEADER")
                .body("string")
                .build()
        );
    }
}